### PR TITLE
CLDC-2773 Update lettings (and sales) csv date format

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
Changes format of date fields in csv export to exclude timestamps and just be YYYY-MM-DD

### CODE REVIEW

File: app/services/csv/sales_log_csv_service.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@ def value(attribute, log)
         value = log.send(attribute)
         get_label(value, attribute, log)
       elsif DATE_FIELDS.include? attribute
-        log.send(attribute)&.iso8601
```
Comment (natdeanlewissoftwire) [inline app/services/csv/sales_log_csv_service.rb:93]: I'm not sure of why this wasn't already public_send given the DATE_FIELDS are only created_at and updated_at, both of which are accessible by public_send so I've changed this

File: spec/fixtures/files/lettings_log_csv_export_non_support_labels.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
...reg,reservist,preg_occ,housingneeds,housingneeds_type,housingneeds_a,housingneeds_b,housingneeds_c,housingneeds_f,housingneeds_g,housingneeds_h,housingneeds_other,illness,illness_type_4,illness_type_5,illness_type_2,illness_type_6,illness_type_7,illness_type_3,illness_type_9,illness_type_8,illness_type_1,illness_type_10,layear,waityear,reason,reasonother,prevten,homeless,ppcodenk,ppostcode_full,prevloc_label,reasonpref,rp_homeless,rp_insan_unsat,rp_medwel,rp_hardship,rp_dontknow,cbl,cap,chr,referral,referral_value_check,incref,earnings,incfreq,hb,has_benefits,benefits,household_charge,nocharge,period,chcharge,wchchrg,carehome_charges_value_check,brent,scharge,pscharge,supcharg,tcharge,scharge_value_check,pscharge_value_check,supcharg_value_check,hbrentshortfall,tshortfall,scheme_code,scheme_service_name,scheme_sensitive,SCHTYPE,scheme_registered_under_care_act,scheme_owning_organisation_name,scheme_primary_client_group,scheme_has_other_client_group,scheme_secondary_client_group,scheme_support_type,scheme_intended_stay,scheme_created_at,location_code,location_postcode,location_name,location_units,location_type_of_unit,location_mobility_type,location_admin_district,location_startdate
```
Comment (kosiakkatrina) [inline spec/fixtures/files/lettings_log_csv_export_non_support_labels.csv:1]: Does this need a new line at the end of the file?

File: app/services/csv/sales_log_csv_service.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/services/csv/sales_log_csv_service.rb:93]: Wouldn't we want to show the time of created at tho?

Comment (natdeanlewissoftwire) [reply app/services/csv/sales_log_csv_service.rb:93]: Ah great point, I thought it didn't specify in the ticket but it actually does - users are having issues with excel when there are timestamps but doesn't seem like it's an issue for created/updated_at, will change

File: spec/fixtures/files/lettings_log_csv_export_non_support_labels.csv (hunk 2 above)
Comment (natdeanlewissoftwire) [reply spec/fixtures/files/lettings_log_csv_export_non_support_labels.csv:1]: not sure why it's showing up as removed in the diff, I think it still has one

Comment (natdeanlewissoftwire) [reply spec/fixtures/files/lettings_log_csv_export_non_support_labels.csv:1]: seems to have gone now I've fixed the merge conflicts 🤷

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**